benchmarks/sindex.py

```diff
@@ -35,8 +35,8 @@ def time_index_creation(self, tree_geom_type):
         Note: pygeos will only create the index once; this benchmark
         is not intended to be used to compare rtree and pygeos.
         """
-        self.data[tree_geom_type]._invalidate_sindex()
-        self.data[tree_geom_type]._generate_sindex()
+        self.data[tree_geom_type].geometry.values._sindex = None
```

Comment:
Should I create create `_invalidate_sindex()`/`_generate_sindex()` function in `array`? Or are we okay with this because it's internal anyway?